## 04_gcc43_build_fix.diff by Brian Nelson <pyro@debian.org>
##
## Fix for GCC 4.3 builds as reported in #452310
--- a/common/string_map.hpp
+++ b/common/string_map.hpp
@@ -11,7 +11,7 @@
 #include "parm_string.hpp"
 #include "posib_err.hpp"
 #include "string_pair.hpp"
-#include "hash.hpp"
+#include "hash-t.hpp"
 #include "objstack.hpp"
 
 
--- a/modules/speller/default/affix.hpp
+++ b/modules/speller/default/affix.hpp
@@ -109,7 +109,7 @@
     }
     WordAff * expand_suffix(ParmString word, const unsigned char * aff,
                             ObjStack &, int limit = INT_MAX,
-                            unsigned char * new_aff = 0, WordAff * * * l = 0,
+                            unsigned char * new_aff_2 = 0, WordAff * * * l = 0,
                             ParmString orig_word = 0) const;
     
   private:
